src/esm_runscripts/filedicts.py

```diff
@@ -1231,12 +1233,19 @@ def execute_filesystem_operation(
         for sim_file_id, sim_file_obj in self.items():
             logger.info(f"Processing {sim_file_id}")
             if config["general"]["jobtype"] == "prepcompute":
-                src, dest = "pool", "work"
+                src, dest = "computer", "work"
             elif config["general"]["jobtype"] == "tidy":
                 src, dest = "work", "exp_tree"
             else:
                 raise ValueError(f"Incorrect jobtype specified for {sim_file_obj}")
             sim_file_obj["_filesystem_op"](src, dest)
+            config[sim_file_obj["component"]]["files"][sim_file_id]["src"] = (
+                sim_file_obj.paths[src]
+            )
+            config[sim_file_obj["component"]]["files"][sim_file_id]["intermediate"] = None
+            config[sim_file_obj["component"]]["files"][sim_file_id]["dest"] = (
+                sim_file_obj.paths[dest]
+            )
```

Comment:
I'm not 100% sure about this, it's a design thing...

what do you think of the following. At this point:
```python
config[sim_file_obj["component"]]["files"][sim_file_id]
```
we have the `SimulationFile`. Should it know about the current phase of movement it is in, and potentially, if it needs an intermediate location?

I would say no, that job belongs somewhere else. Therefore that info also belongs somewhere else...here you are injecting extra info in the `SimulationFile`'s dictionary, right? ...????

Maybe I am overthinking it.